packages/embed/src/css/popover.scss

```diff
@@ -46,6 +46,7 @@
     overflow: hidden;
     line-height: 0;
     border: none;
+    outline: none;
```

Comment:
This is bad for accessibility is there an alternative?

Reply:
the JIRA task says to remove the outline, do you have any other alternative?

Reply:
https://jira.typeform.tf/browse/DIST-899

Reply:
I'd keep it and not remove it all, if needed there are some hacks to add an outline with border-radius